.circleci/config.yml

```diff
@@ -180,19 +179,34 @@ jobs:
 workflows:
   open_new_pr:
     jobs:
-      - install_dependencies
+      - install_dependencies:
+          filters:
+            branches:
+              ignore: main
       - linters:
+          filters:
+            branches:
+              ignore: main
```

Comment:
Can we use the same logic we used on the PR one to only specify the branch filter once and then rely on the `requires` for it to block everything else?

Reply:
I'm pretty confident it'll work. To be brutally honest, I'm more confident it'll work than I was that the previous one would work 😉 and we went ahead with that for science...